indexer/src/main/java/au/org/aodn/esindexer/configuration/WebMvcConfig.java

```diff
@@ -1,25 +1,41 @@
 package au.org.aodn.esindexer.configuration;
 
+import au.org.aodn.ardcvocabs.service.ArdcVocabsService;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.http.MediaType;
 import org.springframework.http.converter.HttpMessageConverter;
 import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.scheduling.annotation.EnableScheduling;
 import org.springframework.web.client.RestTemplate;
 
 import javax.annotation.PostConstruct;
 import java.util.Arrays;
 
 @Configuration
+@ComponentScan("au.org.aodn.ardcvocabs")
+@EnableCaching
+@EnableScheduling
 public class WebMvcConfig {
 
     @Autowired
     protected ObjectMapper objectMapper;
 
+    @Autowired
+    protected ArdcVocabsService ardcVocabsService;
+
+    @Bean
+    public ConcurrentMapCacheManager cacheManager() {
+        return new ConcurrentMapCacheManager(AppConstants.AODN_DISCOVERY_CATEGORIES_CACHE);
+    }
+
```

Comment:
There is a RestTempalte create here again, no crash with the one in ardc?

Reply:
as commented in previous section, no crash with `@Qualifier`:

https://github.com/aodn/es-indexer/pull/63#discussion_r1571896920

Reply:
https://github.com/aodn/es-indexer/pull/63#discussion_r1572160837